packages/office-ui-fabric-react/src/components/TextField/examples/TextField.Styled.Example.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import { ITextFieldStyleProps, ITextFieldStyles, TextField } from 'office-ui-fabric-react/lib/TextField';
+import { ILabelStyles, ILabelStyleProps } from 'office-ui-fabric-react/lib/Label';
+import './TextField.Examples.scss';
```

Comment:
do we need these sass files? If you add a new example, ideally we wouldn't depend on sass. (If you do need them, then it's ok to not change this if it means less changes.)

Reply:
The new example explicitly has JS and CSS styling, so this is needed for the CSS styling example.